.github/workflows/centos.yml

```diff
@@ -0,0 +1,237 @@
+        uses: actions/download-artifact@v2
+        with:
+          name: centos7_sirius
+          path: ${{ env.SIRIUS_INSTALL_DIR }}
+
+      - name: set Sirius variables
+        run: |
+          cd ${{ env.SIRIUS_INSTALL_DIR }}
+          echo "SIRIUS=$PWD/lib" >> $GITHUB_ENV
+          echo "SIRIUS_CMAKE_DIR=$PWD/cmake" >> $GITHUB_ENV
+
+      - name: Configure ortools
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cmake3 -S . -B build -DBUILD_SAMPLES=OFF -DBUILD_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="build/install" -DBUILD_SHARED_LIBS=${{ matrix.shared }} -DBUILD_DEPS=ON -DUSE_SIRIUS=${{ matrix.sirius }} -Dsirius_solver_DIR="${{ env.SIRIUS_CMAKE_DIR }}" -DUSE_XPRESS=${{ matrix.xprs }} -DXPRESS_ROOT="${{ env.XPRESSDIR }}" -DBUILD_PYTHON=${{ matrix.python }} -DBUILD_JAVA=${{ matrix.java }} -DBUILD_DOTNET=${{ matrix.dotnet }}
+
+      - name: Build ortools Linux
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cmake3 --build build --config Release --target all install -j4
+
+      - name: Prepare ortools install
+        if: ${{ matrix.publish-or == 'ON' }}
+        id: or-install
+        run: |
+          find build/install/bin -type f ! \( -name 'protoc*' -o -name 'scip' -o -name 'fz' \) -delete
+          cd build
+          ls -ltr ./install
+          ARCHIVE_NAME="ortools_install${{ steps.names.outputs.appendix_with_shared }}.zip"
+          ARCHIVE_PATH="$PWD/${ARCHIVE_NAME}"
+          zip -r $ARCHIVE_PATH ./install
+          echo "::set-output name=archive_name::$ARCHIVE_NAME"
+          echo "::set-output name=archive_path::$ARCHIVE_PATH"
+
+      - name: Upload ortools install artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ matrix.publish-or == 'ON' }}
+        with:
+          name: ${{ steps.or-install.outputs.archive_name }}
+          path: ${{ steps.or-install.outputs.archive_path }}
+
+      - name: Publish ortools install asset
+        if:  ${{ github.event_name == 'release' && github.event.action == 'created' && matrix.publish-or == 'ON' }}
+        uses: actions/upload-release-asset@v1.0.2
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ${{ steps.or-install.outputs.archive_path }}
+          asset_name: ${{ steps.or-install.outputs.archive_name }}
+          asset_content_type: application/zip
+
+      - name: Run sirius_interface unittests
+        if: ${{ matrix.sirius == 'ON' }}
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cd build
+          ctest3 -V -C Release -R 'cxx_unittests_sirius_interface'
+      - name: Run xpress_interface unittests
+        if: ${{ matrix.xprs == 'ON' }}
+        run: |
+          source /opt/rh/devtoolset-9/enable
+          cd build
+          ctest3 -V -C Release -R 'cxx_unittests_xpress_interface'
+      - name: Run specific cpp tests
+        run: |
+          source /opt/rh/devtoolset-9/enable
```

Comment:
`ctest3` doesn't seem to be defined by `devtoolset-9`, therefore this line shouldn't be included.